### DISCUSSION TITLE
Faster-Whisper on an RTX-4090?

### DISCUSSION THREAD

**Discussion by finebalancetech** [SYSTRAN/faster-whisper#328] [Q&A]
Hey folks,

I'm wondering if anyone has benchmarked on a RTX-4090, specifically the Large models? 

Thanks,
Mark

**Comment by Ericwang1104** [comment]
I did a rough test on a 4090 graphics card
cpu: 12900k memory: 32G graphics card: msi 4090
25 seconds to convert 10 minutes of audio

**Comment by Ericwang1104** [comment]
use medium model for audio transcription wav format
The average length of 1167 files is about 12 minutes, about 8 hours
